deployment/DB_VERSION

```diff
@@ -1 +1,2 @@
-84
+85
+
```

Comment:
not sure if this extra line will not break the deployment: https://github.com/aeternity/epoch/blob/master/.circleci/config.yml#L158